sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -1776,6 +1776,24 @@ object functions {
   }
 
   /**
+   * Returns string, with the first letter of each word in uppercase,
```

Comment:
does this turns other things to lower case? The current documentation sounds like initcap("ABC") will become "Abc"